### PR TITLE
Make RPCS3 compile in C++2a mode

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -951,7 +951,7 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 
 	// Sliders
 
-	EnhanceSlider(emu_settings::DriverWakeUpDelay, ui->wakeupDelay, ui->wakeupText, tr(u8"%0 µs"));
+	EnhanceSlider(emu_settings::DriverWakeUpDelay, ui->wakeupDelay, ui->wakeupText, tr(reinterpret_cast<const char*>(u8"%0 µs")));
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp]: why is the u8 even necessary

Comment (Nekotekina) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Without u8, it's a "string in whatever encoding compiler finds necessary". Even if file is UTF-8.

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: so it's because of the µ ?

Comment (Megamouse) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: also, I think you could also use QChar() if i'm correct

Comment (Nekotekina) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Yes, idk about QChar

Comment (elad335) [reply rpcs3/rpcs3qt/settings_dialog.cpp]: Can just use some overloaded operator""(x) for it.

**Pull request merged**